src/distrs/tdist.jl

```diff
@@ -22,3 +14,27 @@ function tdistlogpdf(ν::T, x::T) where {T<:Real}
     νp12 = (ν + 1) / 2
     return loggamma(νp12) - (logπ + log(ν)) / 2 - loggamma(ν / 2) - νp12 * log1p(x^2 / ν)
 end
+function tdistcdf(ν::T, x::T) where {T<:Real}
+    if x < 0 && ν < (0x1p-26 * x)^2
+       return -log(abs(x))*ν + log(ν)*muladd(ν, T(.5), -1.) - logbeta(ν/2, T(.5))
+    end
+    q = T(0.5)*beta_inc(ν/2, T(.5), ν/muladd(x, x, ν))[1]
+    return ifelse(q > 0, 1 - q, q)
```

Comment:
```suggestion
    half = one(T) / 2
    if x < 0 && ν < (0x1p-26 * x)^2
        return -log(abs(x)) * ν + log(ν) * muladd(ν, half, -1) - logbeta(ν / 2, half)
    end
    q = half * beta_inc(ν / 2, half, ν / muladd(x, x, ν))[1]
    return ifelse(q > 0, 1 - q, q)
```
Summary of suggested changes:
- Functional:
  - Change `-1.` to `-1` to avoid promotion to `Float64`.
  - Use `one(T) / 2` in place of `T(0.5)` since `T` may not be closed under division, so this ensures that the use of 1/2 matches the type of `ν / 2`. Alternatively, you could use `T(0.5) * ν` in place of `ν / 2` or `1 // 2` in place of `T(0.5)`. (If the latter, you'd just need to ensure there's a method of `beta_inc` that can deal with `Rational`s.)
- Stylistic:
  - The body of the `if` had an errant 3-space indent.
  - The surrounding code in this file uses spaces around `*` and `/` so this should be consistent.

These suggestions apply to the other functions here as well.

Reply:
Thanks a lot for the work! I think in the tdistcdf function, this line 
> return ifelse(q > 0, 1 - q, q)

should be changed to 
> return ifelse(x > 0, 1 - q, q)

Else one gets wrong cdfs for negative data, i.e.:
> tdistcdf(1., 100.)  #0.997
> tdistcdf(1., -100.) #0.997